src/statement.js

```diff
@@ -234,6 +258,19 @@ class Statement {
     return this._buildClause('ORDER BY', result.join(', '));
   }
 
+  _buildCTE() {
```

Comment:
if there is a better / more flexible way to do this, let me knw

Reply:
That's perfect, however would it be ok to add an extra space like so:
```js
return this._buildClause('WITH', queries.join(', ')).trim() + ' ';
```
So there will be a space between the CTE end parenthesis and `SELECT/INSERT/UPDATE/DELETE` statements ?

Reply:
will do

Reply:
fixed